docs/dev/discussions.js

```diff
@@ -0,0 +1,428 @@
+/*
+To get these results, sign in to
+https://docs.github.com/en/graphql/overview/explorer
+and do the following query.
+
+{
+  repository(name: "reach-lang", owner: "reach-sh") {
+    discussions(first: 100) {
+      edges {
+        node {
+          objectID: title
+          url: url
+          title: title
+        }
+      }
+    }
+  }
+}
```

Comment:
This needs to be done automatically. We can not have it be manual

Reply:
[786581b](https://github.com/reach-sh/reach-lang/pull/1113/commits/786581b9769b35184cccfaf16530fbcfb20b19dd)